### PR TITLE
ramips: add support for Linksys EA8100

### PR DESCRIPTION
**Pull request by angelsl**

### CODE REVIEW

File: target/linux/ramips/dts/mt7621_linksys_ea7300-v1.dts [hunk 1]
```diff
@@ -1,6 +1,6 @@
 // SPDX-License-Identifier: GPL-2.0-or-later OR MIT
 
-#include "mt7621_linksys_ea7xxx.dtsi"
+#include "mt7621_linksys_ea7xxx_ea8100.dtsi"
```
Comment (adschm) [inline target/linux/ramips/dts/mt7621_linksys_ea7300-v1.dts:3]: Personally, I'd have kept the name just with ea7xxx.

File: target/linux/ramips/image/mt7621.mk [hunk 2]
```diff
@@ -797,13 +797,20 @@ endef
...
+  $(Device/linksys_ea7xxx_ea8100)
   DEVICE_MODEL := EA7500
   DEVICE_VARIANT := v2
   LINKSYS_HWNAME := EA7500v2
 endef
 TARGET_DEVICES += linksys_ea7500-v2
 
+define Device/linksys_ea8100
+  $(Device/linksys_ea7xxx_ea8100)
+  DEVICE_MODEL := EA8100
```
Comment (adschm) [inline target/linux/ramips/image/mt7621.mk:809]: Since there appears to be a v2, we should call this one linksys_ea8100-v1 ...

**Comment by adschm** [comment]
I don't think we need the separate patch for uboot-envtools, and would simply merge that one into the main patch. But that's optional.

**Comment by angelsl** [comment]
@adschm Thanks for the review; I've made the changes. The PR is really trivial now..

I didn't know Linksys released a v2. It looks like it's virtually identical to the v1: https://www.linksys.com/sg/support-article?articleNum=316993

image

**Comment by adschm** [comment]
Okay, thanks, I've added the missing entry in base-files/etc/hotplug.d/ieee80211/10_fix_wifi_mac and will merge it this weekend.

**Comment by angelsl** [comment]
Oh, I didn't even notice. No wonder my BSSID was a bit strange.

Thanks!

**Pull request closed without merging**